### PR TITLE
db/remove-dashboard

### PR DESCRIPTION
**Pull request by karolkolodziej**
The `destroy` dashboard did not work correctly.

Todo:
- [x] Fix errors in the console
- [x] Remove the container
- [x] Test

### CODE REVIEW

File: ts/Dashboards/Layout/Layout.ts [hunk 1]
```diff
@@ -274,9 +274,9 @@ class Layout extends GUIElement {
         const layout = this;
 
         // Destroy rows.
-        for (let i = 0, iEnd = layout.rows.length; i < iEnd; ++i) {
-            layout.rows[i].destroy();
-        }
+        layout.rows.forEach(function (row): void {
+            row.destroy();
+        });
```
Comment (bre1470) [inline ts/Dashboards/Layout/Layout.ts:279]: It is slower. Please restore the regular loop.

**Review by bre1470 — APPROVED** [review]
Looks good with exception of the loop code.

File: ts/Dashboards/Layout/Layout.ts (hunk 1 above)
Comment (sebastianbochan) [reply ts/Dashboards/Layout/Layout.ts:279]: I agree that the regular for (in the appropriate declaration - see in other components) is the fastest.

File: ts/Dashboards/Layout/Row.ts [hunk 2]
```diff
@@ -291,13 +291,15 @@ class Row extends GUIElement {
             }
         }
 
-        row.layout.unmountRow(row);
-        const destroyLayout = layout.rows.length === 0;
+        if (row.layout) {
+            row.layout.unmountRow(row);
+            const destroyLayout = layout.rows.length === 0;
```
Comment (sebastianbochan) [inline ts/Dashboards/Layout/Row.ts:296]: Do we need to declare this for one-time use only?

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**